editors/vscode/server/src/main.rs

```diff
@@ -113,6 +113,7 @@ impl LanguageServer for Backend {
     }
 
     async fn did_change_configuration(&self, params: DidChangeConfigurationParams) {
+        debug!("{:?}", &params.settings);
```

Comment:
Do we want to add a lint for this? I've seen this twice already :-(

Reply:
removed

Reply:
This is different with `dbg! ()`, something we want to explicitly add some log to debug language server in production.

Reply:
But this `debug!` is unused indeed.